src/screens/Button.js

```diff
@@ -154,6 +154,9 @@ const ButtonPage = () => (
             The DOM tag or react component to use for the element.
           </Description>
           <GenericAs />
+          <PropertyValue type="element">
```

Comment:
```suggestion
          <PropertyValue type="element">
             <Description>A specific element to use, such as a router element like Link from react-router-dom</Description>
```

Reply:
Not sure if we want to add a description, but this might help clarify the use case.